docs/blog/2019-04-19-gatsby-why-we-write/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Gatsby: Why We Write
+date: 2019-04-19
+author: Sam Bhagwat
+excerpt: "Why does the Gatsby team and community write so much? Six principles that explain our logorrhea."
+---
+
+If you're a Gatsby community member, you may have noticed something: we write _a lot_.
+
+Community members are often surprised by the comprehensiveness of our [documentation](/docs/), and we publish new article on this blog multiple times per week, from both the core team and the community.
+
+https://twitter.com/mattconvente/status/1099706762897342465?s=19
+
+Writing isn't something we just do because we love it (although we do). Writing is core to who we are as a project, a company, and a community.
+
+Writing is core to the success we've enjoyed so far, and it's core to our plans to become the default way to build on the web.
```

Comment:
> it's core to our plans to become the default way to build on the web

This came off a little too strong to me. Can we soften it with "hopes"? "dreams"?

Reply:
Contrary opinion here--I like pieces that have a bit of a stronger take! That _is_ our plan--so I personally am a fan of this!

Reply:
Good point / counterpoint! I'll use "strategy" as that makes it clear this is our ambition.